python/tidy/servo_tidy/tidy.py

```diff
@@ -723,13 +725,32 @@ def parse_config(content):
     config['ignore']['directories'] = map(lambda path: os.path.join(*path.split('/')),
                                           config['ignore']['directories'])
 
+    # Add dict of dir, list of expected ext to config
+    dirs_for_ext = config_file.get("check_dirs_for_exts", {})
```

Comment:
do you mean removing https://github.com/gterzian/servo/blob/efb85e9e0c70e7e85b9d74ba54782c1e98c139cb/python/tidy/servo_tidy/tidy.py#L39 ?